pyproject.toml

```diff
@@ -21,7 +21,8 @@ dependencies = [
     "jsonschema>=3.0",
     "numpy",
     "pandas>=0.18",
-    "toolz"
+    "toolz",
+    "packaging"
```

Comment:
The `Version` class will be really handy going forward. Is `packaging` a pure Python package that should work everywhere?

Reply:
Exactly. Pure Python package, lightweight with a wheel file being ~48KB, maintained by the Python Packaging Authority.